Makefile

```diff
@@ -25,32 +26,30 @@ help:
 
 .PHONY: build
 build:
-	docker-compose -f ./docker-compose.yaml -f ./tests/infra/docker-compose.test.yaml build \
+	docker-compose build \
 		--build-arg userid=${_UID} --build-arg groupid=${_GID}
 
+.PHONY: format
+format:
+	infra/lint.sh black --fix
+	infra/lint.sh isort --fix
+
 .PHONY: lint
 lint:
-	docker-compose -f ./docker-compose.yaml -f ./tests/infra/docker-compose.lint.yaml build \
-		--build-arg userid=${_UID} --build-arg groupid=${_GID} lint
-
+	docker-compose run --rm web infra/lint.sh
 
 .PHONY: shell
 shell:
-	docker-compose -f ./docker-compose.yaml run web
+	docker-compose -f ./docker-compose.yaml run web /bin/sh
```

Comment:
is `-f ./docker-compose.yaml ` superfluous here?

Reply:
https://github.com/mozilla/jira-bugzilla-integration/pull/88/commits/9d71842bd7ff8699ea0b152575c7cc7b63513146